### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by chukwujekwu-code**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: CI pipeline
```
Comment (cappymayor) [inline .github/workflows/ci.yml]: Something doesn't seems right for adding 11 files

**Review by cappymayor — CHANGES_REQUESTED** [review]
Please check very well what you are doing

**Comment by chukwujekwu-code** [comment]
Yeah . I noticed . Would check once I’m home 😩

File: dags/module/utils.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+
+def customer_age():
+    birth_year = 1980
+
+    def calculate_age(birth_year, current_year):
```
Comment (cappymayor) [inline dags/module/utils.py:5]: This is unacceptable, think about maintaining code @Mychail

**Pull request closed without merging**